cime_config/config_grids.xml

```diff
@@ -3332,6 +3351,15 @@
       <map name="OCN2ATM_SMAPNAME">cpl/gridmaps/ne1024pg2/map_oRRS18to6v3_to_ne1024pg2_nco.200212.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne1024np4.pg2" ocn_grid="ICOS10">
+      <!-- 3km atm /  7.5km ocean.  downscale atm->ocn -->
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/ne1024pg2/map_ne1024pg2_to_ICOS10_nco.211018.nc</map>
```

Comment:
Why use nco as opposed to mono maps here? Were the mono maps bad?

Reply:
No, @brhillman . I didn't try mono map. I went for a quicker map generation with nco, as using TempestRemap was expected to take much longer between two high resolution grids. Do you have any concern of using the nco map?